eval/mt_bench/components.py

```diff
@@ -0,0 +1,169 @@
+# type: ignore
+# pylint: disable=no-value-for-parameter,import-outside-toplevel,import-error
+from typing import List, NamedTuple
+from kfp.dsl import component, Input, Output, Artifact, Model, importer
+from utils.consts import PYTHON_IMAGE
+
+EVAL_IMAGE = "quay.io/sallyom/instructlab-ocp:eval"
+
+@component(base_image=EVAL_IMAGE, packages_to_install=["vllm"]))
+def run_mt_bench_op(
+    models_path_prefix: str,
+    models_list: List[str],
+    mt_bench_output: Output[Artifact],
+    merge_system_user_message: bool,
+    max_workers: str = "auto",
+    device: str = None,
+) -> NamedTuple('outputs', best_model=str, best_score=float):
+
+
+    # TODO: Ensure vLLM server is utilizing GPU
+    def launch_vllm_server_background(model_path: str, retries: int = 60, delay: int = 5):
+        import subprocess
+        import time
+        import requests
+
+        command = f"nohup python3.11 -m vllm.entrypoints.openai.api_server --model {model_path} &"
```

Comment:
```suggestion
        command = [sys.executable, "-m", "vllm.entrypoints.openai.api_server", "--model", "{model_path}"]
```
Why do you need the shell thingy? `subprocess.Popen` spawns a child process already. Please don't do this.

Reply:
It will run in the background, that's what `subprocess.Popen` run a process as a child of current main process. :)

Reply:
hence the `sub` in `subprocess` 😄